Client/Wallet/EthereumProviderHelper.swift

```diff
@@ -80,6 +80,12 @@ class EthereumProviderHelper: TabContentScript {
       return
     }
     
+    if message.webView?.url?.isLocal == false,
```

Comment:
I'm not seeing any issues with session restore,the `webView`s url has updated by the time the request is made/received by us. For error pages/reader mode shouldn't be sending dapp requests anyways, so I think this would be fine.